config/999-cache.yaml

```diff
@@ -23,14 +23,3 @@ spec:
   # This is the Go import path for the binary that is containerized
   # and substituted here.
   image: github.com/knative/serving/cmd/queue
----
-apiVersion: caching.internal.knative.dev/v1alpha1
```

Comment:
can you please also explain this change in the description?  this isn't a config map change.

Reply:
I removed this because it was removed from the observability ConfigMap.